.eslintrc.js

```diff
@@ -1,13 +1,44 @@
 "use strict";
 
-const internalFiles = [
-    "**/cli-engine/**/*",
-    "**/init/**/*",
-    "**/linter/**/*",
-    "**/rule-tester/**/*",
-    "**/rules/**/*",
-    "**/source-code/**/*"
-];
+const path = require("path");
+
+const INTERNAL_FILES = {
+    CLI_ENGINE_PATTERN: "lib/cli-engine/**/*",
+    INIT_PATTERN: "lib/init/**/*",
+    LINTER_PATTERN: "lib/linter/**/*",
+    RULE_TESTER_PATTERN: "lib/rule-tester/**/*",
+    RULES_PATTERN: "lib/rules/**/*",
+    SOURCE_CODE_PATTERN: "lib/source-code/**/*"
+};
+
+/**
+ * Resolve an absolute path or glob pattern.
+ * @param {string} pathOrPattern the path or glob pattern.
+ * @returns {string} The resolved path or glob pattern.
+ */
+function resolveAbsPath(pathOrPattern) {
```

Comment:
Maybe resolveAbsolutePath? We’ve generally not abbreviated words in the codebase.